deploy-manage/deploy/deployment-comparison.md

```diff
@@ -3,55 +3,55 @@
 
-## Security features
+## Security
 
 | Feature/capability | Self-managed | Elastic Cloud Hosted | Serverless |
 |-------------------|-------------|--------------------------------|-------------------------|
-| Custom security configurations | Yes | Limited | No |
-| Authentication realms and custom roles | Yes | Yes | No |
-| Audit logging | Yes | Yes | No |
+| [Security configurations](/deploy-manage/security.md) | Full control | Limited control | Limited control |
+| [Authentication realms](/deploy-manage/users-roles.md) | Available | Available | Available, through Elastic Cloud only |
+| [Custom roles](/deploy-manage/users-roles.md) | Available | Available | Available |
+| [Audit logging](/deploy-manage/monitor/logging-configuration/configuring-audit-logs.md) | Available | Available | No |
 
-## Management features
+## Infrastructure and cluster management
 
 | Feature/capability | Self-managed | Elastic Cloud Hosted | Serverless |
 |-------------------|-------------|--------------------------------|-------------------------|
-| Full control over configuration | Yes | Limited | No |
-| Infrastructure flexibility | Yes | No | No |
-| Autoscaling | No | Yes | Yes |
-| Data tiers management | No | Yes | No |
-| Snapshot management | No | Yes | No |
-| High availability and disaster recovery | Yes | Yes | Yes |
-| Multi-cloud support | No | Yes | Yes |
-| Shard management and replicas | Yes | Yes | No |
+| Hosting | Any infrastructure | Elastic Cloud through AWS, Azure, or GCP | Elastic Cloud through AWS or Azure |
+| Hardware configuration | Full control | Limited control | Managed by Elastic |
+| Autoscaling | No | Available | Automatic |
+| Data tiers management | Through ILM policies | Available | No data tiers |
+| Snapshot management | Custom | Available | Managed by Elastic |
+| High availability and disaster recovery | Available | Available | Managed by Elastic |
+| Shard management and replicas | Available | Available | Managed by Elastic |
 
-## Monitoring features
+## Monitoring
 
 | Feature/capability | Self-managed | Elastic Cloud Hosted | Serverless |
 |-------------------|-------------|--------------------------------|-------------------------|
-| Watcher | Yes | Yes | No |
+| [Deployment health monitoring](/deploy-manage/monitor.md) | Monitoring cluster | AutoOps or monitoring cluster | Managed by Elastic |
+| [Alerting](/explore-analyze/alerts-cases.md) | Watcher or Kibana alerts | Watcher or Kibana alerts | Alerts |
 
-## Data lifecycle features
+## Data lifecycle
 
 | Feature/capability | Self-managed | Elastic Cloud Hosted | Serverless |
 |-------------------|-------------|--------------------------------|-------------------------|
-| Index lifecycle management (ILM) | Yes | Yes | No (uses data streams) |
-| Data tiers management | No | Yes | No |
-| Snapshot management | No | Yes | No |
+| [Data lifecycle management](/manage-data/lifecycle.md) | ILM | ILM and data tiers | Data stream lifecycle |
```

Comment:
ILM, data tiers, and data stream lifecycle are all available to self-managed. 

data stream lifecycle is also available for ECH

Reply:
let's link to explanations for the serverless ones ... treatment tbd

```suggestion
| [Data lifecycle management](/manage-data/lifecycle.md) | ILM | ILM and data tiers | Data stream lifecycle ([why?](/deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings.md#elasticsearch-differences-serverless-features-replaced)) |
```

Reply:
> ILM, data tiers, and data stream lifecycle are all available to self-managed.

> data stream lifecycle is also available for ECH

True... I hate these tables 😭